### DISCUSSION TITLE
Plugin name and button links for "AddSpendTime" wrong

### DISCUSSION THREAD

**Discussion by QuantumQuacken** [aljawaid/PluginManager#66] [General]
Plugin under Manual Plugins - [Repo link](https://github.com/Tagirijus/AddSpentTime/)
The name of the plugin is actually "AddSpen**t**Time", not "AddSpen**d**Time"
The readme and repo links indicated contain the same error in the URL, therefore GitHub returns a 404.

**Comment by aljawaid** [comment]
Hello, this is not actually an issues of the plugin, but the directory file which is updated and then gets pulled by PluginManager, even though Manual Plugins are exclusively displayed in PM.

I originally listed this to help @Tagirijus out so I will fix the errors your mentioned no issues.  

@Tagirijus I will update the manual plugin listing - you need to update the readme as it also contains the spelling typo. Or add me to the repo and I will do it.  You should transform your plugins for general release now to showcase your hardwork. Manual Plugins do not bring the same confidence for users as general plugins, and not everybody uses PluginManager.

**Reply by Tagirijus** [reply]
Thanks for fixing. As soon as I find time to learn how I could solve this thing, I would do it ... I mean it's just forking, fixing, comitting and then doing a pull request, right? :D ... some day I might find the motivation and time!

**Comment by aljawaid** [comment]
@Tagirijus do you want me to switch your manual plugins to general plugins?

**Reply by Tagirijus** [reply]
I'd say: no, since I am updating some of my plugins quite often in the past, since I improve my workflow with my plugins. And then I do not want you to have even more effort to do, in case they should be kept as up to date as possible. I should take a look into doing this whole PR thingy on my on at some point. :D ... at least I managed to almost always keep my releases up to date.

But thanks for asking and thanks that you took action here! (=

**Comment by aljawaid** [comment]
@QuantumQuacken Fixes will show in PM, once the pull request is merged here: https://github.com/kanboard/website/pull/458

**Comment by QuantumQuacken** [comment]
Hello! Thank you for explaining and sorry for disturbing you with unnecessary issues!

**Comment by Tagirijus** [comment]
I did reply on some comments here. Sorry for being so late; I had some kind of forced day off yesterday. Have a nice one, everybody! (=
